src/assets/scss/material.palettes.scss

```diff
@@ -68,7 +68,7 @@ $orcid-red: (
   200: #ef9a9a,
   300: #e57373,
   400: #ef5350,
-  500: #d32f2f,
+  500: #f44336,
   600: #e53935,
   700: #d32f2f,
```

Comment:
@DanielPalafox Let's make sure for now on we overwrite the error colors to use the 700 warn color